lblod/frontend-gelinkt-notuleren
# Feat/full rb copy

### Overview
Added copy screen to rb

##### connected issues and PRs:
GN-5300


### Setup
None

### How to test/reproduce
Go to regulatory attachments, notice you can now click on file options, copy parts, and the copy screen works as expected

### Challenges/uncertainties
In the config, the parts needs to be defined in order of specifity, didn't know how to solve this



### Checks PR readiness
